docker/build-android-with-docker.sh

```diff
@@ -20,7 +20,7 @@ function multiline_helper {
 function get_default_lunch_target {
     local def_lunch_target=""
     case "$(uname -m)" in
-        x86_64|i386|x86|amd64) def_lunch_target="aosp_cf_x86_64_phone-userdebug" ;;
+        x86_64|i386|x86|amd64) def_lunch_target="aosp_cf_x86_64_phone-trunk_staging-userdebug" ;;
         aarch64|arm64) def_lunch_target="aosp_cf_arm64_phone-userdebug" ;;
```

Comment:
shouldn't the arm target have been fixed too?

Reply:
@retroinspect PTAL

Reply:
> shouldn't the arm target have been fixed too?

aosp-main seems to not have cuttlefish arm64 target .
What branch should be used for arm64 target?

Reply:
@jemoreira PTAL